next.config.js

```diff
@@ -135,7 +135,7 @@ module.exports = withSentryConfig(
     silent: true,
 
     org: "opensauced",
-    project: "insights",
+    project: process.env.NODE_ENV === "production" ? "app-opensauced" : "beta-app-opensauced",
```

Comment:
Does this exclude the local development environment or is it considered beta? Just asking as traffic from local devs machines' caused a bunch of noise before.

Reply:
Yes, local development is excluded by default